### PR TITLE
Repositioned the properties in the resource file.

### PR DESCRIPTION
**Pull request by daisukenishino**
- [Change the order of items.](https://github.com/OpenTouryoProject/OpenTouryo/issues/139#issuecomment-178272799)
- [And, change the position of the controls to fit the captions on the screen.](https://github.com/OpenTouryoProject/OpenTouryo/issues/139#issuecomment-178547194)

### CODE REVIEW

**Comment by daisukenishino** [comment]
@SandeepNayak077 

This pullrequest will be merged,
after additional pullrequest have been merged for fix the following problem.

incorrect : "テーブルメンテナンス画面を生成しま"
▼ ▼ ▼
correct : "テーブルメンテナンス画面を生成します。"
```XML
<data name="cbxTableMaintenance.Text" xml:space="preserve">
  <value>テーブルメンテナンス画面を生成します。</value>
</data>
```

**Comment by daisukenishino** [comment]
@yusukemaegawa 

Everything is ready now, sir.
https://github.com/SymphonyTeleca/OpenTouryo/pull/111

Please merge this pullrequest.

File: root/programs/C#/Frameworks/Tools/DaoGen_Tool/Form2.ja-JP.resx [hunk 1]
```diff
@@ -117,15 +117,18 @@
...
-  <data name="Close_ToolStripMenuItem.Text" xml:space="preserve">
-    <value>閉じる </value>
+  <data name="cbxOnlyTableMaintenance.Size" type="System.Drawing.Size, System.Drawing">
+    <value>383, 17</value>
+  </data>
+  <data name="$this.Text" xml:space="preserve">
+    <value>棟梁：D層自動生成ツール（墨壺） - D層、Dao・SQLファイル生成 </value>
+  </data>
+  <data name="btnDaoAndSqlGen.Text" xml:space="preserve">
+    <value>－ Dao・SQL、DTOファイルを生成する －1</value>
```
Comment (yusukemaegawa) [inline root/programs/C#/Frameworks/Tools/DaoGen_Tool/Form2.ja-JP.resx]: @daisukenishino 

What do you mean by "1" of the end?

Comment (daisukenishino) [inline root/programs/C#/Frameworks/Tools/DaoGen_Tool/Form2.ja-JP.resx]: @yusukemaegawa 

Unknown.
Maybe, I think that it changed as follows sometime.
```TXT
－ Dao・SQL、DTOファイルを生成する －
```
▼ ▼ ▼
```TXT
－ Dao・SQL、DTOファイルを生成する －1
```

English translation also have a problem.
https://github.com/SymphonyTeleca/OpenTouryo/blob/nishino/root/programs/C%23/Frameworks/Tools/DaoGen_Tool/Form1.resx#L498

I want to correspond to this at the UI Improvement that is currently being proposed.

##### History
- [Changed the captions Form2.ja-JP.resx and Form2.resx in DaoGen_Tool](https://github.com/SymphonyTeleca/OpenTouryo/commit/c9f868b3c254b6906cc75500198d9fd07776dc2b)
  - [Form2.ja-JP.resx](https://github.com/SymphonyTeleca/OpenTouryo/blob/c9f868b3c254b6906cc75500198d9fd07776dc2b/root/programs/C%23/Frameworks/Tools/DaoGen_Tool/Form2.ja-JP.resx)
- [ツール類の国際化対応(Internationalization of tools.)](https://github.com/SymphonyTeleca/OpenTouryo/commit/f7e8d7c0bb17041467a59ae5ce62caac9bd7f5bf)
  - [Form2.ja-JP.resx](https://github.com/SymphonyTeleca/OpenTouryo/blob/f7e8d7c0bb17041467a59ae5ce62caac9bd7f5bf/root/programs/C%23/Frameworks/Tools/DPQuery_Tool/Form2.ja-JP.resx)

Comment (yusukemaegawa) [inline root/programs/C#/Frameworks/Tools/DaoGen_Tool/Form2.ja-JP.resx]: @daisukenishino 

>Maybe, I think that it changed as follows sometime.

I think that what you should do is only deleting "1" of the end.
So, I think that you can deal with the above in this pull request.

>```TXT
>－ Dao・SQL、DTOファイルを生成する －
>```
>▼ ▼ ▼
>```TXT
>－ Dao・SQL、DTOファイルを生成する －1
>```

I think that the direction is the wrong way around.
```TXT
－ Dao・SQL、DTOファイルを生成する －1
```
▼ ▼ ▼
```TXT
－ Dao・SQL、DTOファイルを生成する －
```

>English translation also have a problem

What do you think the **problem**?

**Comment by daisukenishino** [comment]
@yusukemaegawa 

I erased only "1".
https://github.com/SymphonyTeleca/OpenTouryo/blob/nishino/root/programs/C%23/Frameworks/Tools/DaoGen_Tool/Form2.ja-JP.resx#L131

I think that is reflected in the "Files changed" for a while.
After that, We will correspond this at "UI improvement".

**Pull request merged**